### PR TITLE
Issue #4165: Split and Organize Checkstyle inputs by Test for checks in coding package: PackageDeclaration

### PR DESCRIPTION
**Pull request by Kietzmann**
#4165

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package com.puppycrawl.tools.checkstyle.checks.coding;
 
-class InputPackageDeclaration {
+public class InputPackageDeclarationDiffDirectoryAtParent {
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java]: why is `public` added?

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
     public String value;
-    
+
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java]: please undo superfluous changes.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java (hunk 1 above)
Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java]: It was added by Intellij auto format tool when renaming using ctrl+f6
Fixed

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java (hunk 2 above)
Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java]: Fixed

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java (hunk 1 above)
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java]: in future we should add public to input, as we already split inputs, so we do not expect that smb else require code in this form.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java (hunk 2 above)
Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java]: @romani 
Should I remove unnecessary spacing in such way as it was done previously?

Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java]: No, let us merge this PR asap

**Comment by romani** [comment]
please rebase on latest code to resolve TC problem, TC problems are resolved in our master.

**Comment by Kietzmann** [comment]
@romani 
Done

**Pull request merged**